README.md

```diff
@@ -16,6 +16,14 @@ Then, add an `.eslintrc.json` file with the following:
 
 Then, try running like `npx eslint .`
 
+# Philosophy
+
+Originally, Scoop's eslint configuration was designed to support both Scoop's style guidelines as well as catch syntactical errors. After some evolution, [Prettier](https://prettier.io/) was adopted by Scoop's Frontend
+team, and then eventually expanded to our backend code as well. Our goal is to have our eslint catch all of our syntactical issues, while letting Prettier decide our formatting decisions. No eslint style rule should ever
```

Comment:
Minor: I find `syntactical` and `formatting` to be pretty synonymous. In order to make it a bit clearer about the intent, I suggest we adopt the language used by Prettier and include an example to drive the point home that eslint is meant to catch bugs and prettier is meant to format.

```suggestion
team, and then eventually expanded to our backend code as well. Our goal is to have our eslint catch all of our code-quality issues (`no-unused-vars`), while letting Prettier decide our formatting decisions (`max-len`). No eslint style rule should ever
```